[100%] Testing the TCP connection...
Test project /mnt/i/DIY/CS144/sponge/build
        Start   1: t_wrapping_ints_cmp
  1/159 Test   #1: t_wrapping_ints_cmp ..............   Passed    0.02 sec
        Start   2: t_wrapping_ints_unwrap
  2/159 Test   #2: t_wrapping_ints_unwrap ...........   Passed    0.04 sec
        Start   3: t_wrapping_ints_wrap
  3/159 Test   #3: t_wrapping_ints_wrap .............   Passed    0.02 sec
        Start   4: t_recv_connect
  4/159 Test   #4: t_recv_connect ...................   Passed    0.03 sec
        Start   5: t_recv_transmit
  5/159 Test   #5: t_recv_transmit ..................   Passed    0.10 sec
        Start   6: t_recv_window
  6/159 Test   #6: t_recv_window ....................   Passed    0.02 sec
        Start   7: t_recv_reorder
  7/159 Test   #7: t_recv_reorder ...................   Passed    0.04 sec
        Start   8: t_recv_close
  8/159 Test   #8: t_recv_close .....................   Passed    0.02 sec
        Start   9: t_send_connect
  9/159 Test   #9: t_send_connect ...................   Passed    0.03 sec
        Start  10: t_send_transmit
 10/159 Test  #10: t_send_transmit ..................   Passed    0.07 sec
        Start  11: t_send_retx
 11/159 Test  #11: t_send_retx ......................   Passed    0.02 sec
        Start  12: t_send_window
 12/159 Test  #12: t_send_window ....................   Passed    0.04 sec
        Start  13: t_send_ack
 13/159 Test  #13: t_send_ack .......................   Passed    0.02 sec
        Start  14: t_send_close
 14/159 Test  #14: t_send_close .....................   Passed    0.02 sec
        Start  15: t_strm_reassem_single
 15/159 Test  #15: t_strm_reassem_single ............   Passed    0.02 sec
        Start  16: t_strm_reassem_seq
 16/159 Test  #16: t_strm_reassem_seq ...............   Passed    0.02 sec
        Start  17: t_strm_reassem_dup
 17/159 Test  #17: t_strm_reassem_dup ...............   Passed    0.02 sec
        Start  18: t_strm_reassem_holes
 18/159 Test  #18: t_strm_reassem_holes .............   Passed    0.02 sec
        Start  19: t_strm_reassem_many
 19/159 Test  #19: t_strm_reassem_many ..............   Passed    0.85 sec
        Start  20: t_strm_reassem_overlapping
 20/159 Test  #20: t_strm_reassem_overlapping .......   Passed    0.02 sec
        Start  21: t_strm_reassem_win
 21/159 Test  #21: t_strm_reassem_win ...............   Passed    0.84 sec
        Start  22: t_strm_reassem_cap
 22/159 Test  #22: t_strm_reassem_cap ...............   Passed    0.19 sec
        Start  23: t_byte_stream_construction
 23/159 Test  #23: t_byte_stream_construction .......   Passed    0.03 sec
        Start  24: t_byte_stream_one_write
 24/159 Test  #24: t_byte_stream_one_write ..........   Passed    0.03 sec
        Start  25: t_byte_stream_two_writes
 25/159 Test  #25: t_byte_stream_two_writes .........   Passed    0.02 sec
        Start  26: t_byte_stream_capacity
 26/159 Test  #26: t_byte_stream_capacity ...........   Passed    0.58 sec
        Start  27: t_byte_stream_many_writes
 27/159 Test  #27: t_byte_stream_many_writes ........   Passed    0.02 sec
        Start  28: t_webget
 28/159 Test  #28: t_webget .........................   Passed    1.54 sec
        Start  31: t_tcp_parser
 29/159 Test  #31: t_tcp_parser .....................   Passed    0.03 sec
        Start  32: t_ipv4_parser
 30/159 Test  #32: t_ipv4_parser ....................   Passed    0.08 sec
        Start  33: t_active_close
 31/159 Test  #33: t_active_close ...................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2,)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      close
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=1,ackno=1,seqno=1,)
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=2,win=137) with no payload

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  34: t_passive_close
 32/159 Test  #34: t_passive_close ..................***Failed    0.04 sec
Test Failure on expectation:
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2,)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=1,win=137) with no payload

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  36: t_ack_rst
 33/159 Test  #36: t_ack_rst ........................***Failed    0.04 sec
Test 1
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  38: t_ack_rst_win
 34/159 Test  #38: t_ack_rst_win ....................***Failed    0.03 sec
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  40: t_connect
 35/159 Test  #40: t_connect ........................   Passed    0.03 sec
        Start  42: t_listen
 36/159 Test  #42: t_listen .........................   Passed    0.03 sec
        Start  43: t_winsize
 37/159 Test  #43: t_winsize ........................***Failed    0.04 sec
Test Failure on expectation:
	Action:      write (100248 bytes with 100248 accepted) [f5, 2f, 20, af, 6c, f, f9, d2, 79, 82, a1, d5, d9, 1b, a0, d4, ...]

Failure message:
	100248 bytes should have been written but 0 were

List of steps that executed successfully:
	Action:      listen
	Action:      packet arrives: Header(flags=S,seqno=842785626,ack=0,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,ackno=842785627,win=19778,)
	Action:      packet arrives: Header(flags=A,seqno=842785627,ack=2453079373,win=25062) with no payload
	Expectation: no (more) segments sent
	Expectation: TCP in state sender=`stream ongoing`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1

Warning: Unclean shutdown of TCPConnection
100248 bytes should have been written but 0 were

        Start  45: t_retx
 38/159 Test  #45: t_retx ...........................***Failed    0.03 sec
  check_segment
Test Failure on expectation:
	Expectation: segment sent with (A=1,payload_size=4,data="asdf",)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=400688363,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=400688363,ack=400688364,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=400688364,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not (in check_segment called from line 36)

        Start  46: t_retx_win
 39/159 Test  #46: t_retx_win .......................***Failed    0.03 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2325531451,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2325531451,ack=2325531452,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2325531452,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass
	Action:      20ms pass
	Action:      write (4 bytes) [71, 77, 65, 72]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()s

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  47: t_loopback
 40/159 Test  #47: t_loopback .......................***Failed    0.03 sec
Test Failure on expectation:
	Expectation: TCP has 5025 bytes in flight

Failure message:
	The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `5025`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=1045716749,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=1045716749,ack=1045716750,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1045716750,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=1045716750,ack=1045716750,win=65000) with no payload
	Action:      write (5025 bytes) [71, 9, c5, e6, 1f, 47, 4b, e5, 73, 37, cb, 3f, 30, 2e, 1c, 4, ...]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `5025`

        Start  48: t_loopback_win
 41/159 Test  #48: t_loopback_win ...................***Failed    0.02 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=3525414980,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=3525414980,ack=3525414981,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=3525414981,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=3525414981,ack=3525414981,win=65000) with no payload
	Action:      write (2425 bytes) [a9, 36, 8, 85, d3, 8d, ec, 8e, 91, d3, b3, 53, ee, 6, 37, 19, ...]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  49: t_reorder
 42/159 Test  #49: t_reorder ........................***Failed    0.02 sec
Test Failure on expectation:
	segment sent with (A=1,)
Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2275166884,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=3115592261,ack=2275166885,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=3115592262,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=3115623237,ack=2275166885,win=137) with 180 data bytes "aW<245>U+<154>L<194>R<237><186>/I<26><154><250>..."

Note:
	test 1 failed: no ACK for datagram

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  50: t_address_dt
 43/159 Test  #50: t_address_dt .....................   Passed    0.04 sec
        Start  51: t_parser_dt
 44/159 Test  #51: t_parser_dt ......................   Passed    0.02 sec
        Start  52: t_socket_dt
 45/159 Test  #52: t_socket_dt ......................   Passed    0.04 sec
        Start  53: t_udp_client_send
 46/159 Test  #53: t_udp_client_send ................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25231... done.
new connection from 169.254.144.1:53681.
DEBUG: Outbound stream to 169.254.144.1:53681 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:25231 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
DEBUG: Waiting for clean shutdown... terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20201 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  54: t_udp_server_send
 47/159 Test  #54: t_udp_server_send ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31251... done.
new connection from 169.254.144.1:48403.
DEBUG: Outbound stream to 169.254.144.1:31251 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:48403 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  55: t_udp_client_recv
 48/159 Test  #55: t_udp_client_recv ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:18025... done.
new connection from 169.254.144.1:44129.
DEBUG: Outbound stream to 169.254.144.1:18025 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:44129 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  56: t_udp_server_recv
 49/159 Test  #56: t_udp_server_recv ................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:19599... done.
new connection from 169.254.144.1:55751.
DEBUG: Outbound stream to 169.254.144.1:55751 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20260 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  57: t_udp_client_dupl
 50/159 Test  #57: t_udp_client_dupl ................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:14287... done.
new connection from 169.254.144.1:43399.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20279 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20277 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  58: t_udp_server_dupl
 51/159 Test  #58: t_udp_server_dupl ................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:15448... done.
new connection from 169.254.144.1:57460.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20298 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20296 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  59: t_ucS_1M_32k
 52/159 Test  #59: t_ucS_1M_32k .....................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:8954... done.
new connection from 169.254.144.1:40298.
DEBUG: Outbound stream to 169.254.144.1:40298 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:8954 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20318 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  60: t_ucS_128K_8K
 53/159 Test  #60: t_ucS_128K_8K ....................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32936... done.
new connection from 169.254.144.1:37750.
DEBUG: Outbound stream to 169.254.144.1:37750 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:32936 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20338 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  61: t_ucS_16_1
 54/159 Test  #61: t_ucS_16_1 .......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:2861... done.
new connection from 169.254.144.1:37832.
DEBUG: Outbound stream to 169.254.144.1:37832 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:2861 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20358 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  62: t_ucS_32K_d
 55/159 Test  #62: t_ucS_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32420... done.
new connection from 169.254.144.1:55401.
DEBUG: Outbound stream to 169.254.144.1:55401 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:32420 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20376 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  63: t_ucR_1M_32k
 56/159 Test  #63: t_ucR_1M_32k .....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:17652... done.
new connection from 169.254.144.1:58404.
DEBUG: Outbound stream to 169.254.144.1:17652 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:58404 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  64: t_ucR_128K_8K
 57/159 Test  #64: t_ucR_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:13866... done.
new connection from 169.254.144.1:43015.
DEBUG: Outbound stream to 169.254.144.1:13866 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:43015 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  65: t_ucR_16_1
 58/159 Test  #65: t_ucR_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:27350... done.
new connection from 169.254.144.1:50063.
DEBUG: Outbound stream to 169.254.144.1:27350 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:50063 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  66: t_ucR_32K_d
 59/159 Test  #66: t_ucR_32K_d ......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32109... done.
new connection from 169.254.144.1:54607.
DEBUG: Outbound stream to 169.254.144.1:32109 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:54607 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
DEBUG: Waiting for clean shutdown... terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  67: t_ucD_1M_32k
 60/159 Test  #67: t_ucD_1M_32k .....................***Failed    0.25 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:20819... done.
new connection from 169.254.144.1:36504.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'terminate called after throwing an instance of 'std::exceptionstd::exception'
'
  what():    what():  std::exceptionstd::exception

/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20471 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20469 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  68: t_ucD_128K_8K
 61/159 Test  #68: t_ucD_128K_8K ....................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25122... done.
new connection from 169.254.144.1:34682.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20492 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20490 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  69: t_ucD_16_1
 62/159 Test  #69: t_ucD_16_1 .......................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:16785... done.
new connection from 169.254.144.1:39484.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: terminate called after throwing an instance of 'TCPConnection::write() accepted less than advertised lengthstd::exception
'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20513 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20511 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  70: t_ucD_32K_d
 63/159 Test  #70: t_ucD_32K_d ......................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:23651... done.
new connection from 169.254.144.1:55990.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20532 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20530 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  71: t_usS_1M_32k
 64/159 Test  #71: t_usS_1M_32k .....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:33078... done.
new connection from 169.254.144.1:44954.
DEBUG: Outbound stream to 169.254.144.1:33078 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:44954 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  72: t_usS_128K_8K
 65/159 Test  #72: t_usS_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:13848... done.
new connection from 169.254.144.1:33809.
DEBUG: Outbound stream to 169.254.144.1:13848 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:33809 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  73: t_usS_16_1
 66/159 Test  #73: t_usS_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:6950... done.
new connection from 169.254.144.1:38760.
DEBUG: Outbound stream to 169.254.144.1:6950 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:38760 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  74: t_usS_32K_d
 67/159 Test  #74: t_usS_32K_d ......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:21048... done.
new connection from 169.254.144.1:54294.
DEBUG: Outbound stream to 169.254.144.1:21048 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:54294 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'DEBUG: Waiting for clean shutdown... std::exception'
  what():  std::exception

        Start  75: t_usR_1M_32k
 68/159 Test  #75: t_usR_1M_32k .....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:17978... done.
new connection from 169.254.144.1:50256.
DEBUG: Outbound stream to 169.254.144.1:50256 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:17978 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20626 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  76: t_usR_128K_8K
 69/159 Test  #76: t_usR_128K_8K ....................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:13646... done.
new connection from 169.254.144.1:53776.
DEBUG: Outbound stream to 169.254.144.1:53776 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:13646 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20646 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  77: t_usR_16_1
 70/159 Test  #77: t_usR_16_1 .......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:8421... done.
new connection from 169.254.144.1:56072.
DEBUG: Outbound stream to 169.254.144.1:56072 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20666 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  78: t_usR_32K_d
 71/159 Test  #78: t_usR_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29409... done.
new connection from 169.254.144.1:56697.
DEBUG: Outbound stream to 169.254.144.1:56697 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:29409 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
DEBUG: Waiting for clean shutdown...   what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20684 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  79: t_usD_1M_32k
 72/159 Test  #79: t_usD_1M_32k .....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:7024... done.
new connection from 169.254.144.1:33145.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20705 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20703 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  80: t_usD_128K_8K
 73/159 Test  #80: t_usD_128K_8K ....................***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:28628... done.
new connection from 169.254.144.1:34369.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20726 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20724 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  81: t_usD_16_1
 74/159 Test  #81: t_usD_16_1 .......................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:33088... done.
new connection from 169.254.144.1:40886.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20747 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20745 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  82: t_usD_32K_d
 75/159 Test  #82: t_usD_32K_d ......................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32278... done.
new connection from 169.254.144.1:33481.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20766 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20764 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  83: t_ucS_128K_8K_l
 76/159 Test  #83: t_ucS_128K_8K_l ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:21161... done.
new connection from 169.254.144.1:42848.
DEBUG: Outbound stream to 169.254.144.1:42848 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:21161 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20786 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  84: t_ucS_128K_8K_L
 77/159 Test  #84: t_ucS_128K_8K_L ..................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32667... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20806 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  85: t_ucS_128K_8K_lL
 78/159 Test  #85: t_ucS_128K_8K_lL .................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:1768... done.
new connection from 169.254.144.1:35579.
DEBUG: Outbound stream to 169.254.144.1:35579 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:1768 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20825 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  86: t_ucR_128K_8K_l
 79/159 Test  #86: t_ucR_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25137... done.
new connection from 169.254.144.1:39200.
DEBUG: Outbound stream to 169.254.144.1:25137 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:39200 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  87: t_ucR_128K_8K_L
 80/159 Test  #87: t_ucR_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:21175... done.
new connection from 169.254.144.1:35690.
DEBUG: Outbound stream to 169.254.144.1:21175 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:35690 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  88: t_ucR_128K_8K_lL
 81/159 Test  #88: t_ucR_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:18765... done.
new connection from 169.254.144.1:51200.
DEBUG: Outbound stream to 169.254.144.1:18765 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:51200 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  89: t_ucD_128K_8K_l
 82/159 Test  #89: t_ucD_128K_8K_l ..................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:26666... done.
new connection from 169.254.144.1:39804.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20903 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20901 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  90: t_ucD_128K_8K_L
 83/159 Test  #90: t_ucD_128K_8K_L ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25750... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20924 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  91: t_ucD_128K_8K_lL
 84/159 Test  #91: t_ucD_128K_8K_lL .................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11745... done.
new connection from 169.254.144.1:43697.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: terminate called after throwing an instance of 'std::exception'
  what():  std::exception
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 20944 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 20942 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  92: t_usS_128K_8K_l
 85/159 Test  #92: t_usS_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:21876... done.
new connection from 169.254.144.1:45401.
DEBUG: Outbound stream to 169.254.144.1:21876 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:45401 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  93: t_usS_128K_8K_L
 86/159 Test  #93: t_usS_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:18145... done.
new connection from 169.254.144.1:57494.
DEBUG: Outbound stream to 169.254.144.1:18145 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:57494 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  94: t_usS_128K_8K_lL
 87/159 Test  #94: t_usS_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:23702... 

        Start  95: t_usR_128K_8K_l
 88/159 Test  #95: t_usR_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:28791... done.
new connection from 169.254.144.1:52222.
DEBUG: Outbound stream to 169.254.144.1:52222 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:28791 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21019 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  96: t_usR_128K_8K_L
 89/159 Test  #96: t_usR_128K_8K_L ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11433... done.
new connection from 169.254.144.1:40069.
DEBUG: Outbound stream to 169.254.144.1:40069 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:11433 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21039 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  97: t_usR_128K_8K_lL
 90/159 Test  #97: t_usR_128K_8K_lL .................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:12226... done.
new connection from 169.254.144.1:34240.
DEBUG: Outbound stream to 169.254.144.1:34240 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:12226 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21059 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  98: t_usD_128K_8K_l
 91/159 Test  #98: t_usD_128K_8K_l ..................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:14709... done.
new connection from 169.254.144.1:55582.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21080 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21078 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  99: t_usD_128K_8K_L
 92/159 Test  #99: t_usD_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:17736... done.
new connection from 169.254.144.1:58828.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21101 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21099 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 100: t_usD_128K_8K_lL
 93/159 Test #100: t_usD_128K_8K_lL .................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:27227... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21122 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 101: t_ipv4_client_send
 94/159 Test #101: t_ipv4_client_send ...............***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:9610... done.
new connection from 169.254.144.9:59884.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  DEBUG: Outbound stream to std::exception
169.254.144.9:59884 finished (0 bytes still in flight).
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21139 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 102: t_ipv4_server_send
 95/159 Test #102: t_ipv4_server_send ...............***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:13084... done.
DEBUG: Outbound stream to 169.254.144.9:13084 finished (0 bytes still in flight).
new connection from 169.254.145.9:30091.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 103: t_ipv4_client_recv
 96/159 Test #103: t_ipv4_client_recv ...............***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:30072... done.
new connection from 169.254.144.9:39256DEBUG: Outbound stream to .
169.254.145.9:30072 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 104: t_ipv4_server_recv
 97/159 Test #104: t_ipv4_server_recv ...............***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:11624... done.
new connection from 169.254.145.9:64500.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: Outbound stream to 169.254.145.9:64500 finished (0 bytes still in flight).
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21191 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 105: t_ipv4_client_dupl
 98/159 Test #105: t_ipv4_client_dupl ...............***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:23408... done.
new connection from 169.254.144.9:39509.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21210 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21208 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 106: t_ipv4_server_dupl
 99/159 Test #106: t_ipv4_server_dupl ...............***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:2460... done.
new connection from 169.254.145.9:58858.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21229 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21227 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 107: t_icS_1M_32k
100/159 Test #107: t_icS_1M_32k .....................***Failed    0.27 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:13134... done.
new connection from 169.254.144.9:37062.
DEBUG: Outbound stream to 169.254.144.9:37062 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:13134 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21249 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 108: t_icS_128K_8K
101/159 Test #108: t_icS_128K_8K ....................***Failed    0.28 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:32253... done.
new connection from 169.254.144.9:55341.
DEBUG: Outbound stream to 169.254.144.9:55341 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21269 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 109: t_icS_16_1
102/159 Test #109: t_icS_16_1 .......................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:32306... done.
new connection from 169.254.144.9:5706.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: Outbound stream to 169.254.144.9:5706 finished (0 bytes still in flight).
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21289 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 110: t_icS_32K_d
103/159 Test #110: t_icS_32K_d ......................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:30417... done.
new connection from 169.254.144.9:12211.
DEBUG: Outbound stream to 169.254.144.9:12211 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21307 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 111: t_icR_1M_32k
104/159 Test #111: t_icR_1M_32k .....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:23183... done.
DEBUG: Outbound stream to new connection from 169.254.144.9:7824.
169.254.145.9:23183 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:7824 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 112: t_icR_128K_8K
105/159 Test #112: t_icR_128K_8K ....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:4878... done.
DEBUG: Outbound stream to 169.254.145.9:4878 finished (0 bytes still in flight).

        Start 113: t_icR_16_1
106/159 Test #113: t_icR_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:23167... done.
new connection from 169.254.144.9:30069.
DEBUG: Outbound stream to 169.254.145.9:23167 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 114: t_icR_32K_d
107/159 Test #114: t_icR_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:20714... done.
new connection from DEBUG: Outbound stream to 169.254.145.9:20714 finished (0 bytes still in flight).
169.254.144.9:45530.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 115: t_icD_1M_32k
108/159 Test #115: t_icD_1M_32k .....................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:8660... done.
new connection from 169.254.144.9:40651.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21415 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21413 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 116: t_icD_128K_8K
109/159 Test #116: t_icD_128K_8K ....................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:26564... done.
new connection from 169.254.144.9:36555.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21434 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21436 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 117: t_icD_16_1
110/159 Test #117: t_icD_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:10903... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
new connection from 169.254.144.9:1632.
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21457 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21455 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 118: t_icD_32K_d
111/159 Test #118: t_icD_32K_d ......................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:28202... done.
new connection from 169.254.144.9:25439.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21476 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21474 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 119: t_isS_1M_32k
112/159 Test #119: t_isS_1M_32k .....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:21005... done.
DEBUG: Outbound stream to 169.254.144.9:21005 finished (0 bytes still in flight).
new connection from 169.254.145.9:14847.
DEBUG: Inbound stream from 169.254.145.9:14847 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 120: t_isS_128K_8K
113/159 Test #120: t_isS_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:4626... done.
new connection from 169.254.145.9:5006.
DEBUG: Outbound stream to 169.254.144.9:4626 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:5006 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 121: t_isS_16_1
114/159 Test #121: t_isS_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:10773... done.
DEBUG: Outbound stream to new connection from 169.254.145.9:28119.
169.254.144.9:10773 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 122: t_isS_32K_d
115/159 Test #122: t_isS_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:33381... done.
DEBUG: Outbound stream to 169.254.144.9:33381 finished (0 bytes still in flight).
new connection from 169.254.145.9:26299.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 123: t_isR_1M_32k
116/159 Test #123: t_isR_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:14326... done.
new connection from 169.254.145.9:6499.
DEBUG: Outbound stream to 169.254.145.9:6499 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:14326 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21574 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 124: t_isR_128K_8K
117/159 Test #124: t_isR_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:33253... done.
new connection from 169.254.145.9:39136.
DEBUG: Outbound stream to 169.254.145.9:39136 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:33253 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21594 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 125: t_isR_16_1
118/159 Test #125: t_isR_16_1 .......................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:16205... done.
new connection from 169.254.145.9:41744.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: Outbound stream to 169.254.145.9:41744 finished (0 bytes still in flight).
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21614 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 126: t_isR_32K_d
119/159 Test #126: t_isR_32K_d ......................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:32497... done.
new connection from 169.254.145.9:42354.
DEBUG: Outbound stream to 169.254.145.9:42354 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21632 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 127: t_isD_1M_32k
120/159 Test #127: t_isD_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:32475... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21653 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 128: t_isD_128K_8K
121/159 Test #128: t_isD_128K_8K ....................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:26671... done.
new connection from 169.254.145.9:51512.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21673 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21671 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 129: t_isD_16_1
122/159 Test #129: t_isD_16_1 .......................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:22581... done.
new connection from Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
169.254.145.9:54327.
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21694 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21692 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 130: t_isD_32K_d
123/159 Test #130: t_isD_32K_d ......................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:9197... done.
new connection from 169.254.145.9:24367.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21713 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21711 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 131: t_icS_128K_8K_l
124/159 Test #131: t_icS_128K_8K_l ..................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:8251... done.
new connection from 169.254.144.9:45597.
DEBUG: Outbound stream to 169.254.144.9:45597 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21733 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 132: t_icS_128K_8K_L
125/159 Test #132: t_icS_128K_8K_L ..................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:29187... done.
new connection from 169.254.144.9:7644.
DEBUG: Outbound stream to 169.254.144.9:7644 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:29187 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21753 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 133: t_icS_128K_8K_lL
126/159 Test #133: t_icS_128K_8K_lL .................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:31727... done.
new connection from 169.254.144.9:28903.
DEBUG: Outbound stream to 169.254.144.9:28903 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:31727 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21773 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 134: t_icR_128K_8K_l
127/159 Test #134: t_icR_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:22422... done.
new connection from 169.254.144.9:49454.
DEBUG: Outbound stream to 169.254.145.9:22422 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:49454 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 135: t_icR_128K_8K_L
128/159 Test #135: t_icR_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:2281... done.
DEBUG: Outbound stream to new connection from 169.254.144.9:58489.
169.254.145.9:2281 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:58489 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 136: t_icR_128K_8K_lL
129/159 Test #136: t_icR_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:22381... done.
DEBUG: Outbound stream to 169.254.145.9:22381 finished (0 bytes still in flight).
new connection from 169.254.144.9:11500.
DEBUG: Inbound stream from 169.254.144.9:11500 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 137: t_icD_128K_8K_l
130/159 Test #137: t_icD_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:8164... done.
new connection from 169.254.144.9:28074.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21851 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21849 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 138: t_icD_128K_8K_L
131/159 Test #138: t_icD_128K_8K_L ..................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:3535... done.
new connection from 169.254.144.9:37308.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21870 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21872 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 139: t_icD_128K_8K_lL
132/159 Test #139: t_icD_128K_8K_lL .................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:22322... done.
new connection from 169.254.144.9:38527.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21893 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 21891 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 140: t_isS_128K_8K_l
133/159 Test #140: t_isS_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:17817... done.
DEBUG: Outbound stream to 169.254.144.9:17817 finished (0 bytes still in flight).
new connection from 169.254.145.9:6910.
DEBUG: Inbound stream from 169.254.145.9:6910 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 141: t_isS_128K_8K_L
134/159 Test #141: t_isS_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:32109... done.
DEBUG: Outbound stream to 169.254.144.9:32109 finished (new connection from 0 bytes still in flight).
169.254.145.9:25870.
DEBUG: Inbound stream from 169.254.145.9:25870 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 142: t_isS_128K_8K_lL
135/159 Test #142: t_isS_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:22534... done.
DEBUG: Outbound stream to 169.254.144.9:22534 finished (0 bytes still in flight).
new connection from 169.254.145.9:33956.
DEBUG: Inbound stream from 169.254.145.9:33956 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 143: t_isR_128K_8K_l
136/159 Test #143: t_isR_128K_8K_l ..................***Failed    0.25 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:15377... done.
new connection from 169.254.145.9:12776.
DEBUG: Outbound stream to 169.254.145.9:12776 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21970 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 144: t_isR_128K_8K_L
137/159 Test #144: t_isR_128K_8K_L ..................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:1985... done.
new connection from 169.254.145.9:43167.
DEBUG: Outbound stream to 169.254.145.9:43167 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 21990 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 145: t_isR_128K_8K_lL
138/159 Test #145: t_isR_128K_8K_lL .................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:10419... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22010 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 146: t_isD_128K_8K_l
139/159 Test #146: t_isD_128K_8K_l ..................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:22392... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22030 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 147: t_isD_128K_8K_L
140/159 Test #147: t_isD_128K_8K_L ..................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:16301... done.
new connection from 169.254.145.9:13589.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22050 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 22048 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 148: t_isD_128K_8K_lL
141/159 Test #148: t_isD_128K_8K_lL .................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:21323... done.
new connection from 169.254.145.9:37251.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22071 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 22069 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 149: t_icnS_128K_8K_l
142/159 Test #149: t_icnS_128K_8K_l .................***Failed    0.25 sec
DEBUG: Connecting to 169.254.144.1:24344... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22091 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 150: t_icnS_128K_8K_L
143/159 Test #150: t_icnS_128K_8K_L .................***Failed    0.27 sec
DEBUG: Connecting to 169.254.144.1:29911... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22110 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 151: t_icnS_128K_8K_lL
144/159 Test #151: t_icnS_128K_8K_lL ................***Failed    0.27 sec
DEBUG: Connecting to 169.254.144.1:9521... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22129 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 152: t_icnR_128K_8K_l
145/159 Test #152: t_icnR_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Connecting to 169.254.144.1:7790... done.
DEBUG: Outbound stream to 169.254.144.1:7790 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:7790 has been fully acknowledged.

        Start 153: t_icnR_128K_8K_L
146/159 Test #153: t_icnR_128K_8K_L .................***Timeout  10.00 sec
DEBUG: Connecting to 169.254.144.1:4513... done.
DEBUG: Outbound stream to 169.254.144.1:4513 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:4513 has been fully acknowledged.

        Start 154: t_icnR_128K_8K_lL
147/159 Test #154: t_icnR_128K_8K_lL ................***Timeout  10.01 sec
DEBUG: Connecting to 169.254.144.1:26372... done.
DEBUG: Outbound stream to 169.254.144.1:26372 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:26372 has been fully acknowledged.

        Start 155: t_icnD_128K_8K_l
148/159 Test #155: t_icnD_128K_8K_l .................***Failed    0.23 sec
DEBUG: Connecting to 169.254.144.1:20504... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22211 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 156: t_icnD_128K_8K_L
149/159 Test #156: t_icnD_128K_8K_L .................***Failed    0.26 sec
DEBUG: Connecting to 169.254.144.1:23052... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22231 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 157: t_icnD_128K_8K_lL
150/159 Test #157: t_icnD_128K_8K_lL ................***Failed    0.23 sec
DEBUG: Connecting to 169.254.144.1:14866... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 22251 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 158: t_isnS_128K_8K_l
151/159 Test #158: t_isnS_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:52934.
DEBUG: Inbound stream from 169.254.144.1:52934 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 159: t_isnS_128K_8K_L
152/159 Test #159: t_isnS_128K_8K_L .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:56840.
DEBUG: Inbound stream from 169.254.144.1:56840 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 160: t_isnS_128K_8K_lL
153/159 Test #160: t_isnS_128K_8K_lL ................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... 

        Start 161: t_isnR_128K_8K_l
154/159 Test #161: t_isnR_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:43268.
DEBUG: Outbound stream to 169.254.144.1:43268 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:43268 has been fully acknowledged.

        Start 162: t_isnR_128K_8K_L
155/159 Test #162: t_isnR_128K_8K_L .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:41906.
DEBUG: Outbound stream to 169.254.144.1:41906 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:41906 has been fully acknowledged.

        Start 163: t_isnR_128K_8K_lL
156/159 Test #163: t_isnR_128K_8K_lL ................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... 

        Start 164: t_isnD_128K_8K_l
157/159 Test #164: t_isnD_128K_8K_l .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... 

        Start 165: t_isnD_128K_8K_L
158/159 Test #165: t_isnD_128K_8K_L .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:56554.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 166: t_isnD_128K_8K_lL
159/159 Test #166: t_isnD_128K_8K_lL ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:38122.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception


22% tests passed, 124 tests failed out of 159

Total Test time (real) = 468.31 sec

The following tests FAILED:
	 33 - t_active_close (Failed)
	 34 - t_passive_close (Failed)
	 36 - t_ack_rst (Failed)
	 38 - t_ack_rst_win (Failed)
	 43 - t_winsize (Failed)
	 45 - t_retx (Failed)
	 46 - t_retx_win (Failed)
	 47 - t_loopback (Failed)
	 48 - t_loopback_win (Failed)
	 49 - t_reorder (Failed)
	 53 - t_udp_client_send (Failed)
	 54 - t_udp_server_send (Timeout)
	 55 - t_udp_client_recv (Timeout)
	 56 - t_udp_server_recv (Failed)
	 57 - t_udp_client_dupl (Failed)
	 58 - t_udp_server_dupl (Failed)
	 59 - t_ucS_1M_32k (Failed)
	 60 - t_ucS_128K_8K (Failed)
	 61 - t_ucS_16_1 (Failed)
	 62 - t_ucS_32K_d (Failed)
	 63 - t_ucR_1M_32k (Timeout)
	 64 - t_ucR_128K_8K (Timeout)
	 65 - t_ucR_16_1 (Timeout)
	 66 - t_ucR_32K_d (Timeout)
	 67 - t_ucD_1M_32k (Failed)
	 68 - t_ucD_128K_8K (Failed)
	 69 - t_ucD_16_1 (Failed)
	 70 - t_ucD_32K_d (Failed)
	 71 - t_usS_1M_32k (Timeout)
	 72 - t_usS_128K_8K (Timeout)
	 73 - t_usS_16_1 (Timeout)
	 74 - t_usS_32K_d (Timeout)
	 75 - t_usR_1M_32k (Failed)
	 76 - t_usR_128K_8K (Failed)
	 77 - t_usR_16_1 (Failed)
	 78 - t_usR_32K_d (Failed)
	 79 - t_usD_1M_32k (Failed)
	 80 - t_usD_128K_8K (Failed)
	 81 - t_usD_16_1 (Failed)
	 82 - t_usD_32K_d (Failed)
	 83 - t_ucS_128K_8K_l (Failed)
	 84 - t_ucS_128K_8K_L (Failed)
	 85 - t_ucS_128K_8K_lL (Failed)
	 86 - t_ucR_128K_8K_l (Timeout)
	 87 - t_ucR_128K_8K_L (Timeout)
	 88 - t_ucR_128K_8K_lL (Timeout)
	 89 - t_ucD_128K_8K_l (Failed)
	 90 - t_ucD_128K_8K_L (Failed)
	 91 - t_ucD_128K_8K_lL (Failed)
	 92 - t_usS_128K_8K_l (Timeout)
	 93 - t_usS_128K_8K_L (Timeout)
	 94 - t_usS_128K_8K_lL (Timeout)
	 95 - t_usR_128K_8K_l (Failed)
	 96 - t_usR_128K_8K_L (Failed)
	 97 - t_usR_128K_8K_lL (Failed)
	 98 - t_usD_128K_8K_l (Failed)
	 99 - t_usD_128K_8K_L (Failed)
	100 - t_usD_128K_8K_lL (Failed)
	101 - t_ipv4_client_send (Failed)
	102 - t_ipv4_server_send (Timeout)
	103 - t_ipv4_client_recv (Timeout)
	104 - t_ipv4_server_recv (Failed)
	105 - t_ipv4_client_dupl (Failed)
	106 - t_ipv4_server_dupl (Failed)
	107 - t_icS_1M_32k (Failed)
	108 - t_icS_128K_8K (Failed)
	109 - t_icS_16_1 (Failed)
	110 - t_icS_32K_d (Failed)
	111 - t_icR_1M_32k (Timeout)
	112 - t_icR_128K_8K (Timeout)
	113 - t_icR_16_1 (Timeout)
	114 - t_icR_32K_d (Timeout)
	115 - t_icD_1M_32k (Failed)
	116 - t_icD_128K_8K (Failed)
	117 - t_icD_16_1 (Failed)
	118 - t_icD_32K_d (Failed)
	119 - t_isS_1M_32k (Timeout)
	120 - t_isS_128K_8K (Timeout)
	121 - t_isS_16_1 (Timeout)
	122 - t_isS_32K_d (Timeout)
	123 - t_isR_1M_32k (Failed)
	124 - t_isR_128K_8K (Failed)
	125 - t_isR_16_1 (Failed)
	126 - t_isR_32K_d (Failed)
	127 - t_isD_1M_32k (Failed)
	128 - t_isD_128K_8K (Failed)
	129 - t_isD_16_1 (Failed)
	130 - t_isD_32K_d (Failed)
	131 - t_icS_128K_8K_l (Failed)
	132 - t_icS_128K_8K_L (Failed)
	133 - t_icS_128K_8K_lL (Failed)
	134 - t_icR_128K_8K_l (Timeout)
	135 - t_icR_128K_8K_L (Timeout)
	136 - t_icR_128K_8K_lL (Timeout)
	137 - t_icD_128K_8K_l (Failed)
	138 - t_icD_128K_8K_L (Failed)
	139 - t_icD_128K_8K_lL (Failed)
	140 - t_isS_128K_8K_l (Timeout)
	141 - t_isS_128K_8K_L (Timeout)
	142 - t_isS_128K_8K_lL (Timeout)
	143 - t_isR_128K_8K_l (Failed)
	144 - t_isR_128K_8K_L (Failed)
	145 - t_isR_128K_8K_lL (Failed)
	146 - t_isD_128K_8K_l (Failed)
	147 - t_isD_128K_8K_L (Failed)
	148 - t_isD_128K_8K_lL (Failed)
	149 - t_icnS_128K_8K_l (Failed)
	150 - t_icnS_128K_8K_L (Failed)
	151 - t_icnS_128K_8K_lL (Failed)
	152 - t_icnR_128K_8K_l (Timeout)
	153 - t_icnR_128K_8K_L (Timeout)
	154 - t_icnR_128K_8K_lL (Timeout)
	155 - t_icnD_128K_8K_l (Failed)
	156 - t_icnD_128K_8K_L (Failed)
	157 - t_icnD_128K_8K_lL (Failed)
	158 - t_isnS_128K_8K_l (Timeout)
	159 - t_isnS_128K_8K_L (Timeout)
	160 - t_isnS_128K_8K_lL (Timeout)
	161 - t_isnR_128K_8K_l (Timeout)
	162 - t_isnR_128K_8K_L (Timeout)
	163 - t_isnR_128K_8K_lL (Timeout)
	164 - t_isnD_128K_8K_l (Timeout)
	165 - t_isnD_128K_8K_L (Timeout)
	166 - t_isnD_128K_8K_lL (Timeout)
Errors while running CTest
CMakeFiles/check_lab4.dir/build.make:57: recipe for target 'CMakeFiles/check_lab4' failed
make[3]: *** [CMakeFiles/check_lab4] Error 8
CMakeFiles/Makefile2:131: recipe for target 'CMakeFiles/check_lab4.dir/all' failed
make[2]: *** [CMakeFiles/check_lab4.dir/all] Error 2
CMakeFiles/Makefile2:138: recipe for target 'CMakeFiles/check_lab4.dir/rule' failed
make[1]: *** [CMakeFiles/check_lab4.dir/rule] Error 2
Makefile:155: recipe for target 'check_lab4' failed
make: *** [check_lab4] Error 2
